nbs/01_datasets.ipynb

```diff
@@ -25,7 +25,16 @@
    "cell_type": "code",
```

Comment:
<p>You should make <code>batch_size</code> as an optional argument.</p><p> </p><p><code></code><code>python</p><p>batch_size: Optional[int] = None # batch_size is deprecated in </code>DataModuleConfigs<code></p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/cfnet/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>